### PR TITLE
Add MIT license

### PR DESCRIPTION
**Pull request by juffalow**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Copyright (c) <year> <copyright holders>
```
Comment (iharsh234) [inline LICENSE]: can you do exactly same as in [here](https://github.com/JMI-CSI/jmi-csi.github.io/blob/master/LICENSE)

**Review by iharsh234 — CHANGES_REQUESTED** [review]
add copy right holders name

**Review by iharsh234 — APPROVED** [review]

**Comment by juffalow** [comment]
Sorry, I had to overlook that. Done.

**Comment by iharsh234** [comment]
Thanks @juffalow 💯

**Pull request merged**